src/Microsoft.Data/DataFrame.IO.cs

```diff
@@ -90,13 +90,14 @@ private static Type MaxKind(Type a, Type b)
         /// <param name="guessRows">number of rows used to guess types</param>
         /// <param name="addIndexColumn">add one column with the row index</param>
         /// <returns>DataFrame</returns>
-        public static DataFrame ReadCsv(string filename,
+        public static DataFrame LoadCsv(string filename,
                                 char separator = ',', bool header = true,
                                 string[] columnNames = null, Type[] dataTypes = null,
                                 int numRows = -1, int guessRows = 10,
                                 bool addIndexColumn = false)
         {
-            return ReadStream(() => new StreamReader(filename),
+            Stream fileStream = new FileStream(filename, FileMode.Open);
```

Comment:
You need to dispose this stream when you are done. So do a `using Stream fileStream = new ....`.